content/information-distribution/node.md

```diff
@@ -54,14 +57,65 @@ the Matrix room to join.
 Once a Matrix room is joined, client implementations **must** retrieve and save
 the [list of
 aliases](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-aliases)
-for this room. In the event of an entry node that's currently being used being
-unreachable, clients implementations and **must** use this list to provide users
-with an updated list of alternative entry nodes and aliases to use. The address
-of the node managing a given alias can be retrieved through a [DNS
-query](https://github.com/matrix-org/synapse/#setting-up-federation). Client
-implementations **can** check for pings in order to filter out the inactive
-nodes from this list.
+for this Matrix room. This can be done by processing all of the `m.room.aliases`
+events in the state of the Matrix room, each of them containing the list of
+aliases defined for a given node. The content for such an event looks like this:
+
+```
+"aliases": [
+    "#alias1:example.com",
+    "#alias2:example.com"
+]
+```
+
+The example above shows the content of the `m.room.aliases` state event for the
+node which uses `example.com` as its server name.
+
+Client implementations **must** process these lists of aliases to generate a map
+mapping all of the entry nodes for this Matrix room with the aliases they
+provide for this room.
+
+### Reaching an entry node
+
+Because the list of aliases only list the nodes' server names, and not their
+FQDN, a node might not be reachable at the address  In order to find the
```

Comment:
> a node might not be reachable at the __alias'__ address __.__

Reply:
Whoops, looks like a sentence I forgot to finish. Fixed in 37187cf.